BraveRewardsExample/ViewController.swift

```diff
@@ -87,12 +87,49 @@ class ViewController: UIViewController {
   func setupRewards() {
     if (useMockLedgerSwitch.isOn) {
       rewards = BraveRewards(configuration: .default, ledgerClass: UIMockLedger.self, adsClass: nil)
-      
       // Simulate visiting a sample url to test publisher verification.
       let url = URL(string: "https://bumpsmack.com")!
       rewards.ledger.publisherActivity(from: url, faviconURL: url, publisherBlob: "")
     } else {
       rewards = BraveRewards(configuration: .default)
+      loadContent()
```

Comment:
Im not sure this (and the methods below) are in the right place. Maybe we should extract this to some sort of test data loader class or something that we link to a button click on the UI?

Reply:
update to separate class